src/js/geolocation.js

```diff
@@ -0,0 +1,60 @@
+const btnPos = document
+    .querySelector(".leftAsideBox__findCity--button")
+    .addEventListener('click', getLocation);
```

Comment:
nie podoba mi sie to tutaj, mysle ze dodawanie listenera i przypisywanie elementu do czegos powinny byc odrebnymi operacjami